iOS/MSMapsModules/MSMapsModules/MSMapGeoJsonParser/MSMapGeoJsonParser.m

```diff
@@ -0,0 +1,159 @@
+//
+//  MSGeoJsonParser.h
+//  MSMapsModules
+//
+//  Created by Elizabeth Bartusiak (t-elbart) on 6/12/20.
+//  Copyright © 2020 Microsoft Corporation.
+//  Licensed under the MIT license.
+//
+
+#import "MSMapGeoJsonParser.h"
+#import <MicrosoftMaps/MicrosoftMaps.h>
+
+@implementation MSMapGeoJsonParser {
+  MSMapElementLayer *layer;
+};
+
++ (MSMapElementLayer *)parse:(NSString *)geojson error:(NSError **)error {
+  if (geojson == (id)[NSNull null] || geojson.length == 0) {
+    *error = [NSError
+        errorWithDomain:
+            @"com.microsoft.modules.MSMapsModules.InvalidArgumentError"
+                   code:-100
+               userInfo:@{
+                 NSLocalizedDescriptionKey : @"Input String cannot be null."
+               }];
+    return nil;
+  }
+
+  MSMapGeoJsonParser *instance = [[MSMapGeoJsonParser alloc] init];
+  return [instance internalParse:geojson error:error];
+}
+
+- (MSMapElementLayer *)internalParse:(NSString *)geojson
+                               error:(NSError **)error {
+  layer = [[MSMapElementLayer alloc] init];
```

Comment:
We had a mix of styling in Beacon for setting apart instance variables. We were prepending them with 'm' (mLayer), adding self (self.layer) or starting them with underscore (_layer). I think that since this is a new code we should stick to one and follow it. Google Objective-C Style Guide suggests to use underscore, but in general in Microsoft we don't use underscores as par of variable names. I am fine with using underscores, self and m prefix as the last one. What the rest of the team thinks?

Reply:
My preference is m prefix, mLayer etc.

Reply:
I also like m prefix for instance variables. I like _ for properties